src/test/shell/bazel/bazel_apple_test.sh

```diff
@@ -28,15 +28,18 @@ fi
 
 function set_up() {
   copy_examples
+  IOS_SDK_VERSION=$(xcrun --sdk iphoneos --show-sdk-version)
```

Comment:
setup_objc_test_support would do that for you so we don't have that code in 2 places.